diff -ruN ../src/contracts/ProofOfReserveExecutorV2.sol contracts/ProofOfReserveExecutorV2.sol
--- ../src/contracts/ProofOfReserveExecutorV2.sol	2025-06-26 18:38:14.831458738 +0300
+++ contracts/ProofOfReserveExecutorV2.sol	2025-06-26 19:53:03.858350587 +0300
@@ -103,7 +103,7 @@
   /**
    * @dev disable borrowing for every asset on the pool.
    */
-  function _disableBorrowing() internal {
+  function _disableBorrowing() internal virtual {
     address[] memory reservesList = POOL.getReservesList();
 
     // disable borrowing for all the reserves on the pool
diff -ruN ../src/.gitignore .gitignore
--- ../src/.gitignore	1970-01-01 02:00:00.000000000 +0200
+++ .gitignore	2025-06-26 18:38:44.996397942 +0300
@@ -0,0 +1,2 @@
+*
+!.gitignore
