diff -ruN ../src/contracts/ProofOfReserveExecutorBase.sol contracts/ProofOfReserveExecutorBase.sol
--- ../src/contracts/ProofOfReserveExecutorBase.sol	2025-05-25 11:28:17.576280968 +0300
+++ contracts/ProofOfReserveExecutorBase.sol	2025-05-25 11:38:54.137168817 +0300
@@ -21,10 +21,10 @@
   IProofOfReserveAggregator internal immutable _proofOfReserveAggregator;
 
   /// @dev the list of the tokens, which total supply we would check against data of the associated proof of reserve feed
-  address[] internal _assets;
+  address[] public _assets;
 
   /// @dev token address = > is it contained in the list
-  mapping(address => bool) internal _assetsState;
+  mapping(address => bool) public _assetsState;
 
   /**
    * @notice Constructor.
diff -ruN ../src/contracts/ProofOfReserveExecutorV2.sol contracts/ProofOfReserveExecutorV2.sol
--- ../src/contracts/ProofOfReserveExecutorV2.sol	2025-05-25 11:28:17.576280968 +0300
+++ contracts/ProofOfReserveExecutorV2.sol	2025-05-25 11:38:54.139168804 +0300
@@ -96,7 +96,7 @@
   /**
    * @dev disable borrowing for every asset on the pool.
    */
-  function _disableBorrowing() internal {
+  function _disableBorrowing() internal virtual {
     address[] memory reservesList = _pool.getReservesList();
 
     // disable borrowing for all the reserves on the pool
diff -ruN ../src/.gitignore .gitignore
--- ../src/.gitignore	1970-01-01 02:00:00.000000000 +0200
+++ .gitignore	2025-05-25 11:55:34.578919374 +0300
@@ -0,0 +1,2 @@
+*
+!.gitignore
